torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1923,6 +1923,12 @@ std::shared_ptr<NCCLComm> ProcessGroupNCCL::getNCCLComm(
   // reset log prefix to include group_desc
   logPrefix_ = createLogPrefix();
 
+#ifdef NCCL_COMM_DESCRIPTION
+  // Pass process group name and description to NCCL communicator
+  std::string commDesc = pg_desc_ + ':' + pg_name_;
+  options_->config.commDesc = strdup(commDesc.c_str());
```

Comment:
Is `options_->config` a config to be passed to NCCL initialization? 
It doesn't seem NCCL config has a `commDesc` field. 
https://github.com/NVIDIA/nccl/blob/ab2b89c4c339bd7f816fbc114a4b05d386b66290/src/nccl.h.in#L53-L65